frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -930,6 +943,23 @@ function ReaderFooter:addToMainMenu(menu_items)
                     UIManager:setDirty(nil, "ui")
                 end,
             },
+            {
+                text = _("Progress bar on separate line"),
+                checked_func = function()
+                    return self.settings.progress_bar_separate_line
+                end,
+                enabled_func = function()
+                    return not self.settings.disable_progress_bar
+                end,
+                callback = function()
+                    self.settings.progress_bar_separate_line = not self.settings.progress_bar_separate_line
+                    self:updateFooterContainer()
```

Comment:
Should these three lines be refactored into one update function of some sort?

Reply:
Done :)